pkg/client/cli/cliutil/exec.go

```diff
@@ -0,0 +1,63 @@
+package cliutil
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"strings"
+
+	//nolint:depguard // TODO: switch this stuff over to dexec
+	"os/exec"
+
+	"github.com/telepresenceio/telepresence/v2/pkg/client/logging"
+)
+
+// Run will run the given executable with given args and env, wait for it to terminate, and return
+// the result. The run will dispatch signals as appropriate for the given platform (SIGTERM and SIGINT on Unix platforms
+// and os.Interrupt on Windows).
+func Run(ctx context.Context, sCmd SafeCobraCommand, exe string, args []string, env map[string]string) error {
+	cmd := exec.CommandContext(ctx, exe, args...)
+	cmd.Stdout = sCmd.OutOrStdout()
+	cmd.Stderr = sCmd.ErrOrStderr()
+	cmd.Stdin = sCmd.InOrStdin()
```

Comment:
Here's something that slipped in recently that you may not have noticed: The tests don't do `cmd.SetOut` or anything anymore; the fork the actual process.  So it shouldn't be necessary to explicitly pass around stdio anymore (whether via a cobra command or via separate stdin/out/err arguments).

Reply:
Not removing them, but just using `cmd.Stdout = os.Stdout` instead; no need to take the `sCmd` argument.

Reply:
OOH okay my bad

Reply:
Also note that if we don't need to pass in `sCmd`, then there's no reason to move SafeCobraCommand in to cliutil.